src/components/classifier/QuestionClassifier.js

```diff
@@ -98,15 +98,13 @@ class QuestionClassifier extends Component {
             isSuccess,
             needsTutorial,
             tutorial,
-            project,
-            workflow,
             subject,
             subjectsSeenThisSession,
             task,
-            inBetaMode,
             guide,
             answers
         } = this.props
+        const { project, workflow, inBetaMode } = this.props.route.params;
```

Comment:
Similar to this line, I think line 73 within `submitClassification` could use `const { workflow } = this.props.route.params;` and workflow can be removed from the props destructuring on line 79.